dist/tsc/paymasterclient.d.ts

```diff
@@ -0,0 +1,58 @@
+import { ethers, FetchRequest, JsonRpcApiProviderOptions, Networkish, TransactionRequest } from 'ethers';
+import type { AddressLike } from 'ethers/src.ts/address';
+import type { BigNumberish } from 'ethers/src.ts/utils';
+export type IsSponsorableResponse = {
+    Sponsorable: boolean;
+    SponsorName: string;
+    SponsorIcon: string;
+    SponsorWebsite: string;
+};
+export declare enum GaslessTransactionStatus {
+    New = 0,
+    Pending = 1,
+    Confirmed = 2,
+    Failed = 3,
+    Invalid = 4
+}
+export type GaslessTransaction = {
+    readonly TxHash: string;
+    readonly BundleID: bigint;
+    readonly FromAddress?: AddressLike;
+    readonly ToAddress?: AddressLike;
+    readonly RawData: string;
+    readonly Status: GaslessTransactionStatus;
+    readonly GasUsed: bigint;
+    readonly PolicyID: bigint;
+    readonly Source: string;
+    readonly BornBlockNumber: bigint;
+    readonly ChainID: number;
+};
+export type SponsorTx = {
+    readonly TxHash: string;
+    readonly Address: AddressLike;
+    readonly BundleUUID: string;
+    readonly Status: GaslessTransactionStatus;
+    readonly GasPrice?: BigNumberish;
+    readonly GasFee?: BigNumberish;
+    readonly BornBlockNumber: bigint;
+    readonly ChainID: number;
+};
+export type Bundle = {
+    readonly BundleUUID: string;
+    readonly Status: GaslessTransactionStatus;
+    readonly AvgGasPrice?: BigNumberish;
+    readonly BornBlockNumber: bigint;
+    readonly ConfirmedBlockNumber: bigint;
+    readonly ConfirmedDate: bigint;
+    readonly ChainID: number;
+};
+export declare class PaymasterClient extends ethers.JsonRpcProvider {
+    constructor(url?: string | FetchRequest, network?: Networkish, options?: JsonRpcApiProviderOptions);
+    ChainID(): Promise<string>;
+    isSponsorable(tx: TransactionRequest): Promise<IsSponsorableResponse>;
+    sendRawTransaction(signedTx: string): Promise<string>;
+    getGaslessTransactionByHash(hash: string): Promise<GaslessTransaction>;
```

Comment:
we have a new API: `eth_GetTransactionCount`, please add it

Reply:
already have